tests/integration_tests/test_wave_04_05_06.py

```diff
@@ -4,8 +4,8 @@
 from swap_meet.decor import Decor
 from swap_meet.electronics import Electronics
 
-@pytest.mark.skip
-@pytest.mark.integration_test
+# @pytest.mark.skip
+# @pytest.mark.integration_test
```

Comment:
Next time remember to only remove the `@pytest.mark.skip` decorator. The second one is supposed to stay.